cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -32,9 +32,15 @@ <h1 class="d-inline"> {{ record.title_additional or record.title }} </h1>
             {% endfor %}
             {% endif %}
         {% endif %}
+        {% if record.categories.primary %} <a class="badge badge-category"  href="/search?category={{ record.categories.primary }}"> {{ record.categories.primary }}</a>
+            {% if record.categories.secondary %}
+              {% for subcategory in record.categories.secondary %}
+                <a class="badge badge-subcategory" href="/search?category={{ record.categories.primary }}&subcategory={{ subcategory }}"> {{ subcategory }}</a>
+              {% endfor %}
+            {% endif %}
+        {% endif %}
         {% if record.experiment %} <a class="badge badge-experiment" href="/search?experiment={{ record.experiment }}">{{ record.experiment }}</a> {% endif %}
-        {% if record.collision_information %} <a class="badge badge-tag" href="/search?q={{ record.collision_information.energy }}">Collision energy {{ record.collision_information.energy }}</a> {% endif %}
-        {% if record.accelerator %} <a class="badge badge-tag"  href="/search?q={{ record.accelerator }}">Accelerator {{ record.accelerator }}</a> {% endif %}
```

Comment:
The accelerator badge does not need the word `accelerator`, in a similar way to the `collision energy badge`